e2e_tests/tests/experiment/test_profiling.py

```diff
@@ -15,12 +15,10 @@
 @pytest.mark.e2e_slurm_gpu
 @pytest.mark.timeout(30 * 60)
 @pytest.mark.parametrize(
-    "model_def,timings_enabled",
-    [
-        (conf.fixtures_path("mnist_pytorch"), True),
-    ],
+    "model_def",
```

Comment:
nit: I don't understand the parameterization. I think it's cleaner without it

Reply:
done